java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -992,55 +992,78 @@ void testStringConcat() {
                                                    "IJ\"\u0100\u0101\u0500\u0501IJ\"\u0100\u0101\u0500\u0501",
                                                    "kl mkl m", "Nop1Nop1", "\\qRs2\\qRs2", "3tuV\'3tuV\'",
                                                    "wX4YzwX4Yz", "\ud720\ud721\ud720\ud721");
-         Scalar emptyString = Scalar.fromString("");
-         ColumnVector concat = ColumnVector.stringConcatenate(emptyString, emptyString,
-                                                              v, v)) {
-      assertColumnsAreEqual(concat, e_concat);
+         Scalar emptyString = Scalar.fromString("")) {
+      ColumnVector[] arr = new ColumnVector[2];
+      arr[0] = v;
+      arr[1] = v;
+      try (ColumnVector concat = ColumnVector.stringConcatenate(emptyString, emptyString, arr)) {
```

Comment:
nit: didn't need to change as much of the tests.  You could have just done 

```
Scalar emptyString = Scalar.fromString("");
 ColumnVector concat = ColumnVector.stringConcatenate(emptyString, emptyString,
                                                              new ColumnVector[]{v, v})) {

```

But I am happy to let this in without the change.

Reply:
While in Java land, the variable arguments accepts an array instead of a comma separated list. When Scala tries to use these apis, it enforces a ColumnVector type check for variable arguments. Changing this to an array allows me to match the type check and work around the error.

Reply:
That is much cleaner, didn't think about that test change. Will clean it up to remove the additional try with resource blocks.